### DISCUSSION TITLE
Show warning symbol on Scratch Addons when signed out

### DISCUSSION THREAD

**Discussion by bobby-sills** [ScratchAddons/ScratchAddons#5746] [Ideas]
I think it would be helpful for Scratch messages to show a warning symbol when someone is signed out of their Scratch account. That way you know if you aren't receiving any messages because you're signed out. Maybe something like this: (nvm, github is being weird, I'll just up a link) https://scratch.mit.edu/projects/807125940/

**Comment by Samq64** [comment]
There's already an option for this in the message count addon.

**Comment by DNin01** [comment]
Do you mean Messages count on extension icon or Scratch Messaging?

**Reply by iqnite** [reply]
https://scratch.mit.edu/scratch-addons-extension/settings#addon-msg-count-badge
